lib/style-observer.js

```diff
@@ -422,7 +421,13 @@ var StyleObserver = (function() { // jshint ignore:line
 
         sheetToFilterSelectorMap = new utils.WeakMap();
         pendingStyles = new utils.Set();
-        ignoredSheets = new utils.Set(sheetsToIgnore);
+        ignoredSheets = new utils.Set();
```

Comment:
It'd be better to use the code from https://github.com/AdguardTeam/ExtendedCss/pull/65

3. [ ] Merge with master and PR 65

Reply:
We can merge this first and then merge PR 65

Reply:
Also possible, whichever you choose, but it needs to be merged with master in either case.